### ISSUE TITLE
Emulator not running.

### ISSUE THREAD

**Issue by cprakashagr** [tddpirate/androiddocker#1]
Hi,

I was trying to run this project. However, I am not able to run the ARM based emulators.

From Android Studio, the emulator UI is coming and then exiting abruptly.
From terminal, I execute it as:
`./emulator -avd Nexus_5X_API_24`

And I get this error,
`QObject::~QObject: Timers cannot be stopped from another thread`

I have tried solutions from [this](http://stackoverflow.com/questions/36554322/cannot-start-emulator-in-android-studio-2-0) link, but I get no luck on that.

**Comment by tddpirate** [comment]
Are you running the most recent version of Android Studio?
Did you perform apt-get upgrade?
Do x86 based emulators work for you?

**Comment by cprakashagr** [comment]
Hi,
x86 emulator work on the systems having the virtualisation enabled. System in which I have tried doesn't have virtualisation. So only arch which I can try is ARM. 

However, usual ARM based emulators start very smoothly, without docker.

**Comment by tddpirate** [comment]
I apologize for not replying earlier.

Since Android emulators (both x86 and ARM) work very slowly on my PC, I use real devices rather than emulators.

However, to try to reproduce your problem, I installed "Install ARM EABI v7a System Image (revision: 7)" and created a Nexus 5X AVD using it.

When trying to start it from Android Studio, the UI came and stayed until I killed the relevant processes (did not wait until end of its boot - would have taken me too long time).

So the reported problem was not reproduced.

The reported error message looks like a Qt library problem. However, I found no Qt packages in the Docker image (using `dpkg -l | egrep -i qt` running inside a shell opened by `docker exec -it android-studio /bin/bash`) so I do not know where and how did Qt creep in.
